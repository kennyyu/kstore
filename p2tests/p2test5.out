(500)
(300)
(400)
(700)
(800)
(600)
(100)
(200)
(900)
(500,50)
(300,30)
(400,40)
(700,70)
(800,80)
(600,60)
(100,10)
(200,20)
(900,90)
(500,5)
(300,3)
(400,4)
(700,7)
(800,8)
(600,6)
(100,1)
(200,2)
(900,9)
(500,50,5)
(300,30,3)
(400,40,4)
(700,70,7)
(800,80,8)
(600,60,6)
(100,10,1)
(200,20,2)
(900,90,9)
(50)
(30)
(40)
(70)
(80)
(60)
(10)
(20)
(90)
(50,500)
(30,300)
(40,400)
(70,700)
(80,800)
(60,600)
(10,100)
(20,200)
(90,900)
(50,5)
(30,3)
(40,4)
(70,7)
(80,8)
(60,6)
(10,1)
(20,2)
(90,9)
(50,500,5)
(30,300,3)
(40,400,4)
(70,700,7)
(80,800,8)
(60,600,6)
(10,100,1)
(20,200,2)
(90,900,9)
