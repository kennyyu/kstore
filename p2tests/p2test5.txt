create(t6a,"sorted")
create(t6b,"unsorted")
create(t6c,"unsorted")
load("p2table6.csv")
tempA=select(t6a)
resAA=fetch(t6a,tempA)
resAB=fetch(t6b,tempA)
resAC=fetch(t6c,tempA)
tuple(resAA)
tuple(resAA,resAB)
tuple(resAA,resAC)
tuple(resAA,resAB,resAC)
tempB=select(t6b)
resBA=fetch(t6a,tempB)
resBB=fetch(t6b,tempB)
resBC=fetch(t6c,tempB)
tuple(resBB)
tuple(resBB,resBA)
tuple(resBB,resBC)
tuple(resBB,resBA,resBC)
