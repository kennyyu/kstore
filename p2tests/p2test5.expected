(100)
(200)
(300)
(400)
(500)
(600)
(700)
(800)
(900)
(100,10)
(200,20)
(300,30)
(400,40)
(500,50)
(600,60)
(700,70)
(800,80)
(900,90)
(100,1)
(200,2)
(300,3)
(400,4)
(500,5)
(600,6)
(700,7)
(800,8)
(900,9)
(100,10,1)
(200,20,2)
(300,30,3)
(400,40,4)
(500,50,5)
(600,60,6)
(700,70,7)
(800,80,8)
(900,90,9)
(10)
(20)
(30)
(40)
(50)
(60)
(70)
(80)
(90)
(10,100)
(20,200)
(30,300)
(40,400)
(50,500)
(60,600)
(70,700)
(80,800)
(90,900)
(10,1)
(20,2)
(30,3)
(40,4)
(50,5)
(60,6)
(70,7)
(80,8)
(90,9)
(10,100,1)
(20,200,2)
(30,300,3)
(40,400,4)
(50,500,5)
(60,600,6)
(70,700,7)
(80,800,8)
(90,900,9)
